From: Gaosheng Cui <cuigaosheng1@huawei.com>
Subject: ia64/kprobes: remove orphan declarations from arch/ia64/include/asm/kprobes.h
Date: Wed, 14 Sep 2022 20:47:07 +0800

flush_register_stack() and invalidate_stacked_regs() have been removed
since commit 0aeaf6b3a345 ("ia64/kprobes: Remove jprobe implementation"),
so remove them.

Link: https://lkml.kernel.org/r/20220914124707.1483471-1-cuigaosheng1@huawei.com
Signed-off-by: Gaosheng Cui <cuigaosheng1@huawei.com>
Cc: Ingo Molnar <mingo@kernel.org>
Cc: Masami Hiramatsu (Google) <mhiramat@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 arch/ia64/include/asm/kprobes.h |    2 --
 1 file changed, 2 deletions(-)

--- a/arch/ia64/include/asm/kprobes.h~ia64-kprobes-remove-orphan-declarations-from-arch-ia64-include-asm-kprobesh
+++ a/arch/ia64/include/asm/kprobes.h
@@ -110,8 +110,6 @@ extern int kprobe_fault_handler(struct p
 extern int kprobe_exceptions_notify(struct notifier_block *self,
 				    unsigned long val, void *data);
 
-extern void invalidate_stacked_regs(void);
-extern void flush_register_stack(void);
 extern void arch_remove_kprobe(struct kprobe *p);
 
 #endif /* CONFIG_KPROBES */
_
